components/team_general_tab/team_general_tab.jsx

```diff
@@ -12,6 +12,8 @@ import SettingItemMax from 'components/setting_item_max.jsx';
 import SettingItemMin from 'components/setting_item_min.jsx';
 import SettingPicture from 'components/setting_picture.jsx';
 
+const ACCEPTED_TEAM_IMAGE_TYPES = ['image/jpeg', 'image/png', 'image/bmp'];
```

Comment:
Should these be moved to constants.jsx? I can't imagine a scenario where we support different types of files for profile pictures and team icons

Reply:
0/5 but I think it's better to be explicit. I'm worried about someone changing the constant/text for profiles but not for team icons